tokens/BAS.json

```diff
@@ -0,0 +1,21 @@
+[{
+	"name": "Toshi Token",
+	"address": "0x8544FE9D190fD7EC52860abBf45088E81Ee24a8c",
+	"symbol": "TOSHI",
+	"decimals": 18,
+	"chainId": 8453,
+	"logoURI": "https://static.debank.com/image/eth_token/logo_url/0xf136d7b0b7ae5b86d21e7b78dfa95375a7360f19/7727b7001e9a63b857bf48afae76da99.png"
```

Comment:
Seems that the logo should be changed? Since https://basescan.org/token/images/toshibase2_32.png is different from current one

Reply:
We got the address from debank, I updated it but both should work.